setup.py

```diff
@@ -1,7 +1,8 @@
 from setuptools import setup, find_packages
+NotFoundError = getattr(__builtins__, 'ModuleNotFoundError', ImportError)
 try:
     import md5  # fix for "No module named _md5" error
-except ImportError:
+except NotFoundError:
```

Comment:
Why not just keep it as `ImportError`? `ModuleNotFoundError` is a subclass of `ImportError` so the current code will still work fine.